oca_dependencies.txt

```diff
@@ -1 +1,2 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+sale-workflow https://github.com/archeti-org/sale-workflow 14.0-mig-sale_order_secondary_unit
```

Comment:
```suggestion
sale-workflow
```

Reply:
Hello! 
I updated the oca_dependencies.txt file as you recommend me but still have this issue:
ERROR: Could not find a version that satisfies the requirement odoo14-addon-sale_order_secondary_unit (from versions: none)